autoload/lsc/message.vim

```diff
@@ -2,6 +2,22 @@ function! lsc#message#show(message, ...) abort
   call s:Echo('echo', a:message, get(a:, 1, 'Log'))
 endfunction
 
+function! lsc#message#showRequest(message, ...) abort
+  let actions = get(a:, 2, [])
+  let options = [a:message]
+  let index = 0
+  while index < len(actions)
+    call add(options, (index + 1) . '. ' . get(actions, index)['title'])
```

Comment:
How about `' - '` as the separator?

That will match https://github.com/natebosch/vim-lsc/blob/3949861c954fa9fec475bd839ed8e88cb6383c06/autoload/lsc/edit.vim#L76